apps/video/test/marionette/play_multiple_types_test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var assert = require('assert'),
+    Video = require('./lib/video');
+
+marionette('play various types of videos', function() {
+
+  var app, client, selectors;
+
+  client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    }
+  });
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    selectors = Video.Selector;
+    app = new Video(client);
+  });
+
+  function verifyVideo(type, done) {
+    var title = 'VID_0001';
```

Comment:
How about changing this and the filenames to "test" or "sample".  That might make it more obvious what the files are for.